src/test_application/activescenes/scene_vehicles.cpp

```diff
@@ -580,38 +979,70 @@ Session setup_test_vehicles(
 
     // Build "PartVehicle"
```

Comment:
Vehicle building code starts here. Basically, this generates a VehicleData struct, which is then read by some tasks elsewhere in this source file that copies it to the scene based on vehicle spawn requests from scenarios.cpp.